pipelines/managed/push-tekton-task-bundles-to-external-registry/push-tekton-task-bundles-to-external-registry.yaml

```diff
@@ -391,9 +386,8 @@ spec:
         - name: taskGitRevision
           value: "$(params.taskGitRevision)"
       runAfter:
-        - collect-registry-token-secret
         - push-snapshot
-    - name: update-trusted-tasks
+    - name: prepare-data-acceptable-bundles
```

Comment:
This one should probably have `runAfter: update-trusted-tasks`, right? We like to have something set explicitly.